docker_mcp/core/transfer/archive.py

```
@@ -247,7 +269,7 @@ async def extract_archive(
 
         result = await asyncio.get_event_loop().run_in_executor(
             None,
-            lambda: subprocess.run(  # noqa: S603
+            lambda: subprocess.run(  # nosec B603  # nosec B603
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)